UIPS/UIP-0131.md

```diff
@@ -0,0 +1,121 @@
+---
+uip: "0131"
+title: "Ministry"
+description: An Azimuth Multibridge
+author: ~master-malwyl, ~rolryx, ~sarlev-sarsen
+status: Draft
+type: Standards Track
+category: PKI
+created: ~2024.11.20
+---
+
+Abstract
+--------
+
+Ministry enables Urbit's Azimuth PKI to operate across multiple blockchain platforms while maintaining Ethereum as its root of trust. The system introduces a secure bridging protocol that allows points to move between supported chains through galaxy-operated oracles. This proposal specifies both the general bridge architecture and a concrete first implementation supporting Coinbase's Base layer-2 rollup.
+
+Motivation
+----------
+
+The Urbit namespace currently exists solely on Ethereum. While Ethereum provides strong property rights guarantees, limiting Azimuth to a single chain restricts user optionality and increases operational costs. As the blockchain ecosystem evolves with new Layer 2s and alternative Layer 1s, there is value in making Urbit IDs portable while preserving both the security guarantees and property rights of the existing system.
```

Comment:
There's a whole swath of Bitcoiners who would otherwise be very urbit aligned if it weren't for the PKI being on Ethereum. At a higher level than that, if Urbit is aiming to be a self-sovereign identity system, being tied to a single consensus layer kneecaps that capacity to pick up users of different socioeconomic networks. Users ought to be able to put their identity on the consensus network that aligns with their needs/value system. The fact that ownership on the Urbit (Arvo/Ames) Network is client side validated suggests (or proves) that we needn't be restricted to on-chain validation or single chain consensus as the incentive for any given Arvo node operator is to maintain an accurate copy of (cross-chain) Azimuth state such that they can communicate with other members of the network.